cmd/src/cli/js-tests-scaffold/index.js

```diff
@@ -1,21 +1,45 @@
 // This test file uses the tape testing framework.
 // To learn more, go here: https://github.com/substack/tape
 const test = require('tape');
-const Container = require('@holochain/holochain-nodejs');
+const { ConfigBuilder, Container } = require('@holochain/holochain-nodejs');
 
-// instantiate an app from the DNA JSON bundle
-const app = Container.instanceFromNameAndDna("app", "dist/bundle.json")
+// IIFE to keep config-only stuff out of test scope
```

Comment:
not sure what IIFE is, better not to have a comment in a scaffold file with some insider language

Reply:
k.  I'll fix that.

Reply:
MDN suggests 'self-executing anonymous function' as an alternative to 'IIFE'. Or could just call it a closure, and people will notice that it's an immediately invoked one.

Reply:
fixed